BookingProject/BookingWebAPI/BookingWebApi/Controllers/ApartamentController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BookingWebApi.Application.DTOs.ApartamentDTOs;
+using BookingWebApi.Application.Extensions;
+using BookingWebApi.Application.Filters;
+using BookingWebApi.Application.Interfaces;
+using BookingWebApi.Helpers;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Cryptography.X509Certificates;
+
+namespace BookingWebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ApartamentController(
+        IApartamentService _apartamentService    
+    ) : ControllerBase
+    {
+        [Authorize(Roles ="Host")]
```

Comment:
Why isn't this role in Constants on Domain layer? Move it there pls

Reply:
Fixed